### ISSUE TITLE
IOUSBDeviceDescription persists through boot.

### ISSUE THREAD

**Issue by coolstar** [comex/safestrat#2]
Continuing boot from safestrat by killing sshd results in the IOUSBDeviceDescription still persisting even after lockdownd and SpringBoard start, which means that iTunes is unable to connect to the device.

**Comment by 0neday** [comment]
@coolstar  Yes, but you can fix it by this approach:   change string "i am a serial number" in the safestrat.c file(line 46) into your own device identifier.

**Comment by coolstar** [comment]
@0neday that's not really a fix. That's a workaround.. and a dirty one at that as it can't be distributed

**Comment by 0neday** [comment]
@coolstar Yeah, your are right! But, maybe you can write a function to get  device identifier, and replace string "i am a serial number" in safestrat.c file. This way could fix distribution problem
